app/src/androidTest/java/com/google/jetpackcamera/FlashDeviceTest.kt

```diff
@@ -74,14 +73,10 @@ internal class FlashDeviceTest {
             .assertExists()
             .performClick()
 
-        // Exit quick settings
-        composeTestRule.onNodeWithTag(QUICK_SETTINGS_DROP_DOWN)
+        composeTestRule.onNodeWithTag(QUICK_SETTINGS_FLASH_BUTTON)
             .assertExists()
-            .performClick()
-
-        assert(
-            UiTestUtil.getPreviewCameraAppSettings(this@runScenarioTest).flashMode ==
-                FlashMode.ON
+        composeTestRule.onNodeWithContentDescription(
+            com.google.jetpackcamera.quicksettings.R.string.quick_settings_flash_on_description
```

Comment:
It looks like these could be combined. The `QUICK_SETTINGS_FLASH_BUTTON` node is what has the content discription, correct?